app/src/test/java/com/example/musicapp/InstrumentDataIntegrationTest.kt

```diff
@@ -13,46 +13,32 @@ import org.junit.Assert.*
 import org.junit.Rule
 import org.junit.rules.TestRule
 
-
-
 class InstrumentDataIntegrationTest {
 
     @get:Rule
-    var rule: TestRule =  InstantTaskExecutorRule()
-    lateinit var mvm:MainViewModel
-
+    var rule: TestRule = InstantTaskExecutorRule()
+    lateinit var mvm: MainViewModel
     var instrumentService = mockk<InstrumentService>()
 
-
     @Test
-    fun TestInstrumentListForResults() {
-
+    internal fun TestInstrumentListForResults() {
         givenAFeedOfInstrumentDataAreAvailable()
         getAllTheInstruments()
         checkIfInstrumentDataIsEmpty()
-
     }
 
     private fun givenAFeedOfInstrumentDataAreAvailable() {
-
```

Comment:
I just removed the unnecessary blank line here (Not one of my code review suggestions).